catalog/resource_catalog.go

```diff
@@ -43,14 +43,30 @@ func (a CatalogsAPI) getCatalog(name string) (ci CatalogInfo, err error) {
 	return
 }
 
-func (a CatalogsAPI) deleteCatalog(name string) error {
-	// TODO: force_destroy attribute
-	return a.client.Delete(a.context, "/unity-catalog/catalogs/"+name+"?force=true", nil)
+func (a CatalogsAPI) deleteCatalog(name string, force bool) error {
```

Comment:
@nkvuong can you create a separate method and call it `forceDelete`? this will align with future SDK migration